internal/cmd/watch.go

```diff
@@ -72,15 +75,56 @@ var watchCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(watchCmd)
-
 	watchCmd.PersistentFlags().IntVarP(&watchCmdSettings.interval, "interval", "i", 2, "interval between measurements in seconds")
+	watchCmd.PersistentFlags().StringVarP(&watchCmdSettings.resource, "resource", "r", "", "resource to watch (default to pods)")
 	watchCmd.PersistentFlags().StringSliceVarP(&watchCmdSettings.namespaces, "namespace", "n", []string{}, "comma separated list of namespaces to filter (default is to use all namespaces")
+	watchCmd.PersistentFlags().StringVarP(&watchCmdSettings.crd, "crd", "c", "", "crd to watch, based on the singular or short-name of the resource")
 }
 
-func printWatchList(hvnr havener.Havener) error {
+func printWatchList(hvnr havener.Havener) (err error) {
+	var out string
+
+	if watchCmdSettings.crd != "" && watchCmdSettings.resource != "" {
```

Comment:
Just for separation concerns, I would move the command flag verification out of the print function and into the `RunE` function.

Reply:
sure, unfortunately cobra does not have a flag dependency management, so I came up with that. Also, this save us of having yet another subcommand for watching crds.